pkg/kube/roundtripper.go

```
@@ -58,10 +58,10 @@ func (rt *RetryingRoundTripper) roundTrip(req *http.Request, retry int, prevResp
 	r.Seek(0, io.SeekStart)
 	resp.Body = io.NopCloser(r)
 	if err != nil {
-		return resp, rtErr
+		return resp, err
 	}
 	if ke.Code < 500 {
-		return resp, rtErr
+		return resp, err
```

This was ignoring errors less than 500, so nil probably